Ddpb 3485 - lifestyle behat

jamesrwarren opened the pull request:
## Purpose
Added the health and welfare lifestyle section. Checked it doesn't appear on NDR or PFA.

![image](https://user-images.githubusercontent.com/58939809/118695872-92130780-b805-11eb-8b43-d01eff9770c7.png)

Fixes DDPB-3485

## Approach
Used the expected results function again. Seemed to save me a lot of time. Think we could maybe genericise some other bits like the  'I Visit' bits but I didn't do much refactoring in this one. Kept it simple and followed existing patterns.

## Learning
I added health and welfare types in... probably could have done all the others I know we'll need but I was just trying to get this small pr done and dusted.. Soz

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [x] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [ ] Translations are used and the profiler doesn't identify any missing
- [ ] Any links or buttons added are screen reader friendly and contextually complete
- [ ] If adding GA events, I have updated or [checked](docs/runbooks/GOOGLE-ANALYTICS.md) the existing category or label values

acsauk reviewed (commented):
All looks good - I think we should add one further scenario for when a user answers both questions and then edits the initial question to no so the form gets cleared. Ready to merge after that 👍

jamesrwarren commented:
Added the extra test for editing the second answer to no. There isn't actually a yes/no for the first question, just different text which we were changing anyway. I added the extra step to change second from yes to no anyway :)

acsauk reviewed (approved):
:shipit:
